command/apply.go

```diff
@@ -400,11 +406,18 @@ func outputsAsString(state *terraform.State) string {
 		for _, k := range keys {
 			v := outputs[k]
 
-			outputBuf.WriteString(fmt.Sprintf(
-				"  %s%s = %s\n",
-				k,
-				strings.Repeat(" ", keyLen-len(k)),
-				v))
+			if schemaMap[k].Sensitive {
```

Comment:
It shouldn't at this stage - apply has already run so the state will be modified.